### ISSUE TITLE
HTTP custom status text/reason lost from Tomcat to IIS

### ISSUE THREAD

**Issue by kwaschny** [Bilal-S/iis2tomcat#102]
We switched from [ISAPI redirector](https://tomcat.apache.org/connectors-doc/webserver_howto/iis.html) to BonCode (ColdFusion) and now no longer see custom HTTP status text/reason in the response.

We specify a custom status response like this:
`<cfheader statusCode="404" statusText="Test">`
(Should be [`HttpServletResponse.sendError()`](https://tomcat.apache.org/tomcat-8.5-doc/servletapi/javax/servlet/http/HttpServletResponse.html#sendError(int,%20java.lang.String)) behind the curtain.)

With the old ISAPI redirector, the status text/reason is delivered to IIS as `404 Test`:
![image](https://user-images.githubusercontent.com/35005943/105887374-f23e2d80-600b-11eb-94bd-cd9c08a58feb.png)
_(Screenshot from Chrome Network/Request-Response View)_

BonCode however, always returns `404 Not Found`. I checked the repo code and was looking for [HttpResponse.StatusDescription](https://docs.microsoft.com/en-us/dotnet/api/system.web.httpresponse.statusdescription), but couldn't find anything about status text/reason at all. Is this simply not supported or is there something wrong with the setup on our side?

**BonCodeAJP13.settings**
```
<Settings>
    <Port>8012</Port>
    <EnableAdobeMode>True</EnableAdobeMode>
</Settings>
```

**web.config**
```
<system.webServer>
    <httpErrors errorMode="Custom" existingResponse="PassThrough" />
</system.webServer>
```

**Comment by Bilal-S** [comment]
@kwaschny 
You are correct. It looks like we do not look for an statusText just the statusCode is passed along.
So this would need to be an enhancement to be made in the future.
